audonnx/core/model.py

```diff
@@ -205,6 +205,7 @@ def __call__(
             sampling_rate: int,
             *,
             outputs: typing.Union[str, typing.Sequence[str]] = None,
+            concat: bool = False,
```

Comment:
Usually I'm in favor of long names, e.g. `sampling_rate` instead of `fs`, but I have to say for `concat` I prefer the shorter version as I can never remember how to write `concatenate` ;)